en/flight_controller/crazyflie2.md

```diff
@@ -17,134 +17,152 @@ The Crazyflie line of micro quads was created by Bitcraze AB. An overview of the
-* [Logitech Joystick](https://www.logitechg.com/en-ch/product/f310-gamepad)
+* [Logitech Joystick](https://www.logitechg.com/en-ch/product/f310-gamepad).
 
 ## Flashing PX4
 
-After setting up the PX4 development environment, follow these steps to put the PX4 software on the CF2:
+After setting up the PX4 development environment, follow these steps to install the PX4 Autopilot on the Crazyflie 2.0:
 
-1. Grab source code of the PX4 [Bootloader](https://github.com/PX4/Bootloader)
-1. Compile it using:
+1. Download the source code of the PX4 Bootloader:
+   ```
+   git clone https://github.com/PX4/Bootloader.git
+   ```
+1. Navigate into the top directory of the source code and compile it using:
    ```
    make crazyflie_bl
    ```
-1. Put the CF2 into DFU mode by following these steps:
+1. Put the Crazyflie 2.0 into DFU mode by following these steps:
    - Ensure it is initially unpowered.
-   - Hold down button.
+   - Hold down the reset button (see figure below...).
    - Plug into computer's USB port.
    - After a second, the blue LED should start blinking and after 5 seconds should start blinking faster.
    - Release button.
-1. Flash bootloader using *dfu-util* and unplug CF2 when done: 
+   
+1. Install *dfu-util*:
+   ```
+   sudo apt-get update
+   sudo apt-get install dfu-util
+   ```
+
+1. Flash bootloader using *dfu-util* and unplug Crazyflie 2.0 when done: 
    ```
    sudo dfu-util -d 0483:df11 -a 0 -s 0x08000000 -D ./build/crazyflie_bl/crazyflie_bl.bin
    ``` 
-   If successful, then the yellow LED should blink when plugging in again
-1. Grab the [Firmware](https://github.com/PX4/Firmware)
-1. Compile it using:
+   When powering on the Crazyflie 2.0 the yellow LED should blink.
+   
+1. Download the source code of the PX4 autopilot:
+   ```
+   git clone https://github.com/PX4/Firmware.git
+   ```
+1. Navigate into the top directory of the source code and compile it using:
    ```
    make crazyflie_default upload
    ```
-1. When prompted to plug in device, plug in CF2. 
+1. When prompted to plug in device, plug in Crazyflie 2.0. 
    The yellow LED should start blinking indicating bootloader mode. 
    Then the red LED should turn on indicating that the flashing process has started.
 1. Wait for completion.
-1. Done! Calibrate via QGC.
+1. Done! Calibrate the sensors using [QGroundControl](https://docs.qgroundcontrol.com/en/SetupView/Sensors.html).
 
-> **Note** If QGC does not connect with the vehicle, ensure that in [nuttx-config](https://github.com/PX4/Firmware/blob/master/platforms/nuttx/nuttx-configs/crazyflie/nsh/defconfig#L934) for crazyflie `# CONFIG_DEV_LOWCONSOLE is not set` is replaced by `CONFIG_DEV_LOWCONSOLE=y`
+> **Note** If QGroundControl does not connect with the vehicle, ensure that in [nuttx-config](https://github.com/PX4/Firmware/blob/master/platforms/nuttx/nuttx-configs/crazyflie/nsh/defconfig#L934) for crazyflie `# CONFIG_DEV_LOWCONSOLE is not set` is replaced by `CONFIG_DEV_LOWCONSOLE=y`
```

Comment:
Can you remove this? It is not needed anymore.

Reply:
@MariliG See above!